code/src/index.css

```diff
@@ -1,13 +1,166 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Chakra Petch", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(10, 37, 255);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.mainContainer {
+  background-color: rgb(255, 236, 234);
+  border: 2px solid rgb(246, 166, 34);
+  width: 320px;
+  margin: 50px auto;
+  padding: 10px;
+}
+
+h1 {
```

Comment:
To write less code you can combine similar styling features for h1, h2, and possible h3 (color, font size and width are the same).